src/js/util.js

```diff
@@ -324,6 +324,11 @@ var Util;
         insertHTMLCommand: function (doc, html) {
             var selection, range, el, fragment, node, lastNode, toReplace;
 
+            // cleanup fragment comment created by Chrome
+            html = html
```

Comment:
Yeah, it should be.
I was a bit in a hurry when debugging this stuff I missed the replacement options part.